db/src/main/java/net/bible/android/database/bookmarks/BookmarksDao.kt

```diff
@@ -45,6 +46,7 @@ CASE WHEN :orderBy = 'BIBLE_ORDER' THEN Bookmark.kjvOrdinalStart END,
 CASE WHEN :orderBy = 'BIBLE_ORDER' THEN Bookmark.startOffset END,
 CASE
     WHEN :orderBy = 'CREATED_AT' THEN Bookmark.createdAt
+    WHEN :orderBy = 'CREATED_AT_DESC' THEN Bookmark.createdAt
```

Comment:
And here I would feel that you meant

```suggestion
    WHEN :orderBy = 'CREATED_AT_DESC' THEN Bookmark.createdAt DESC
```

Or what do you think?